### ISSUE TITLE
Amo Vecchio

### ISSUE THREAD

**Issue by Tetsu93** [fuji97/pokemon-xenoverse-tracker#796]
ho provato a pescare molte volte e in zone diverse ma non trovo mai niente, dice sempre "nemmeno una vecchia ciabatta..."

**Comment by Tetsu93** [comment]
ah, ho capito... è buggato se assegnato a f5, funziona solo se si fa usa

**Comment by Yorunokami** [comment]
Esatto, ci ho messo un po a capirlo anche io. non va l-assegnazione
